modules/claims_api/app/sidekiq/claims_api/special_issue_updater.rb

```diff
@@ -11,6 +12,13 @@ class SpecialIssueUpdater < UpdaterService
     sidekiq_retries_exhausted do |message|
       log_exception_to_claim_record(message['args'].last, { text: 'Job retries exhausted' })
       log_exception_to_sentry(StandardError.new("Failed to apply special issues to contention: #{message}"))
+
+      ClaimsApi::Logger.log(
+        'claims_api_retries_exhausted',
+        claim_id: message['args'].last,
+        detail: "Job retries exhausted for #{message['class']}",
+        error: message['error_message']
+      )
```

Comment:
@FonzMP  The special issues has these two additional logging lines (below for reference) and I was unsure about including all that in with the flash updater.  You are correct though I think it would be a safe move and I certainly can move it in there. I would just want clarification on if we should include or exclude the additional two lines of logging for the Flash Updater.  I think I could use the class value in the message to check so excluding them would be easy or just to change the message, but just wasn't sure about suddenly adding additional logging to the flash updater, _probably_ doesn't really matter though.

```
log_exception_to_claim_record(message['args'].last, { text: 'Job retries exhausted' })
log_exception_to_sentry(StandardError.new("Failed to apply special issues to contention: #{message}"))
```